bot.py

```diff
@@ -65,6 +65,20 @@
 LOG = structlog.get_logger()
 debug = os.environ.get('DEBUG', None) == 'yes'
 
+auth_header = {}
+API_KEY = os.getenv('CUSTOM_API_KEY', None)
+if (API_KEY and API_KEY != 'secret_key'):
+    auth_header = { 'X-API-Key': API_KEY }
+
+DISCORD_KEY = os.getenv('DISCORD_KEY')
+DJANGO_HOST = os.getenv('DJANGO_HOST', 'localhost')
+DJANGO_PORT = int(os.getenv('DJANGO_PORT', 8000))
+GAME_URL = os.getenv('GAME_URL', r'''si.bitcrafter.net''')
```

Comment:
```suggestion
GAME_URL = os.getenv('GAME_URL', 'si.bitcrafter.net')
```

don't think I see a need to use a raw string literal here rather than just a string literal